test/unitTests/observers/telemetryObserver.test.ts

```diff
@@ -69,12 +70,28 @@ describe('TelemetryObserver', () => {
             assert.deepStrictEqual(sentEventProperties, {
                 CorrelationId: 'fakedCorrelationId',
                 Result: 'Failed',
+                RetrievedFromCache: 'false',
                 SignInType: undefined,
                 UserName: undefined,
                 UserEmail: undefined,
                 ErrorCode: 'GitHubSignInFailed',
             });
         });
+
+        it('CredentialRetrieveFromLocalCredentialManager', () => {
```

Comment:
update the test to use `UserSignInSucceeded`

Reply:
it is using `UserSignInSucceeded`

---
In reply to: [384404196](https://github.com/docascode/vscode-docs-build/pull/70#discussion_r384404196) [](ancestors = 384404196)

Reply:
I see